pkg/cmd/server/api/serialization_test.go

```diff
@@ -68,6 +69,11 @@ func fuzzInternalObject(t *testing.T, forVersion unversioned.GroupVersion, item
 				// The final value of OAuthConfig.MasterCA should never be nil
 				obj.OAuthConfig.MasterCA = &s
 			}
+			if time.Now().Second()%2 == 0 {
```

Comment:
You're right.  I tested without this and the serialization test works.  Removed.